src/pages/my-speak/interview/hooks/useSpeechRecognition.ts

```
@@ -253,9 +361,85 @@ export const useSpeechRecognition = (): UseSpeechRecognitionReturn => {
+          resolve();
+        } catch (err) {
+          console.error('[Mobile] Failed to send audio:', err);
+          setError('음성 전송에 실패했습니다. 다시 시도해주세요.');
+          setIsListening(false);
+          resolve();
+        }
+      };
+
+      mediaRecorderRef.current!.stop();
+    });
+  }, [isListening, sessionId, messageType]);
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)